docusaurus/docs-internals/editor-controls/data-controls/motion.md

```diff
@@ -190,3 +254,206 @@ An array of animation names that should be disabled. In this case, you cannot se
+If `vertical` motion is enabled and its `speed` is greater than or equal to 0.5, the `.brz-box` element will translate 
+vertically by that speed value multiplied by 10 pixels.
+Also, if `transparency` level >= 0.5, then we will set `.brz-box` element to display `"flex"`, 
+otherwise it will be displayed `"none"`;
+
+
+```js
+{
+  id: "motion",
+  type: "motion",
+  style: ({ value }) => {
+    const display = value?.transparency?.level >= 0.5 ? "flex" : "none";
+    
+    const horizontalTranslate = value?.horizontal?.enabled && value?.horizontal?.speed >= 0.5
+      ? `translateX(${value.horizontal.speed * 10}px)`
+      : "translateX(0)";
+
+    const verticalTranslate = value?.vertical?.enabled && value?.vertical?.speed >= 0.5
+      ? `translateY(${value.vertical.speed * 10}px)`
+      : "translateY(0)";
+
+    return {
+      "{{WRAPPER}} .brz-text": {
+        transform: horizontalTranslate,
+      },
+      "{{WRAPPER}} .brz-box": {
+        transform: verticalTranslate,
+        display
+      }
+    }           
+  }
+}
+```
+
+#### Usage in HTML example
+
+In this example, we demonstrate the use of `motion` control values (`horizontal` and `rotate`) to dynamically transform and 
+control a carousel component's behavior. The `Carousel` component reacts to changes in `motion` settings to adjust 
+its display, offering a dynamic user experience.
+
+This example demonstrates how `motion` control values can be utilized to create dynamic and responsive user interfaces,
+enhancing the user experience by adjusting visual elements based on user-defined `motion` settings.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import React, { JSX } from "react";
+
+interface Props {
+  active: undefined | "vertical" | "horizontal" | "transparency" | "blur" | "rotate" | "scale" | "mouseTrack" | "mouseTilt";
+  vertical: undefined | { speed: number; direction: "up" | "down"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  horizontal: undefined | { speed: number; direction: "left" | "right"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  transparency: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  blur: undefined | { direction: "in" | "out" | "outIn" | "inOut"; level: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  rotate: undefined | { direction: "left" | "right"; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; speed: number; viewport: { bottom: number; top: number; type: "viewport"; } };
+  scale: undefined | { direction: "up" | "down" | "downUp" | "upDown"; speed: number; x: "left" | "center" | "right"; y: "top" | "center" | "bottom"; viewport: { bottom: number; top: number; type: "viewport"; } };
+  mouseTrack: undefined | { direction: "direct" | "opposite"; speed: number; };
+  mouseTilt: undefined | { direction: "direct" | "opposite"; speed: number; };
```

Comment:
not valid props, don't forget that your props is prefixed by your control's `id`

P.S. as example you can check how the motion keys looks in `Wrapper/defaultValue.json`